docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -300,60 +293,14 @@ to allow duplicate library aliases. A custom validator must be specified for thi
 property to be enabled - the naming scheme’s default validator will not be altered.
 
 # Setting Up the Run Scanner
-The run scanner is a webservice that scans the paths containing
+Run Scanner is a webservice that scans the paths containing
 sequencer output. It is not required for a functioning MISO install, but
 without it, sequencer runs must be added manually.
 
-Run Scanner must be hosted on a separate server from MISO. Create a file called `ROOT.xml`
-in the following directory `$CATALINA_HOME/conf/Catalina/localhost` on that server (create
-the directory if necessary), and populate it with the following information:
-
-    <Context>
-       <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
-    </Context>
-
-In `/etc/runscanner.json`, or another path of your choosing, put JSON data describing your instruments. You will need one record for each instrument:
-
-    {
-      "path": "/some/directory/where/sequencer/writes",
-      "platformType": "ILLUMINA",
-      "name": "default",
-      "timeZone": "America/Toronto",
-      "parameters": {}
-    }
-
-The JSON file then contains a list of instruments:
+Please see the Run Scanner readme for setup instructions.
 
-    [
-      {
-        "path": "/srv/sequencer/hiseq2500_1",
-        "platformType": "ILLUMINA",
-        "name": "default",
-        "timeZone": "America/Toronto",
-        "parameters": {}
-      },
-      {
-        "path": "/srv/sequencer/hiseq2500_2",
-        "platformType": "ILLUMINA",
-        "name": "default",
-        "timeZone": "America/Toronto",
-        "parameters": {}
-      }
-    ]
+Once complete, edit `$CATALINA_HOME/conf/Catalina/localhost/miso.properties` of the MISO Tomcat server and set `miso.runscanner.urls` to the URL of the Run Scanner instance. It is possible to set up multiple run scanners managing different sequencers and add all the URLs to `miso.properties`. Restart MISO.
```

Comment:
"Restart MISO" may case some confusion, as at this point in the guide users haven't yet learned how to start MISO.